service/controller/v11/templates/ignition/cloud_provider_conf.go

```diff
@@ -9,6 +9,7 @@ location: {{ .Location }}
 aadClientId: {{ .AADClientID }}
 aadClientSecret: {{ .AADClientSecret }}
 {{- end}}
+cloudProviderRateLimit: true
```

Comment:
How should this help? Can we have a comment that explains why this is necessary and what it is doing?

Reply:
Don't we need to specify `cloudProviderRateLimitQPSWrite` and `cloudProviderRateLimitQPS`?

Reply:
@xh3b4sd This is a config file that is passed to cloud provider magic on azure nodes (the one that adds labels to nodes and so on). This simply enables the cloud provider implementation here to honor rate limits (as it is set to false by default). Not sure what I should comment here.

@fiunchinho My understanding of the docs was that they are only _valid_ if this flag is set to true, not _required_. I would expect azure to set sensible defaults here, but it wouldn't be the first time I was wrong with that assumption.

Reply:
Put yourself into the shoes of some new PE or SRE who was not involved in all the hustle here before. Then he deals with all kinds of weird flags and config files. What would you like to tell them in the comment? Let's just explain the mechanism a bit and provide reasoning why we need it and what would happen when we would not have this.

Reply:
IMO comment isn't necessary here. This is provider config. You need to google that as all other settings in the file. Putting comments on all of would obscure the config more than help imo.

Reply:
I totally disagree with this. People brag about missing docs all the time and now you refuse to add them. I leave you alone with this now. Disagree and commit.

Reply:
Comment would be good, and a link to an issue with more details.